src/test/java/org/bonitasoft/connectors/ws/SecureWSConnectorTest.java

```diff
@@ -134,25 +119,24 @@ public void testHTTPHeaderOK() throws Exception {
         header.add(headerValue);
         requestHeaders.put(headerName, header);
 
-        execute(request.toString(), SOAPBinding.SOAP11HTTP_BINDING, "http://localhost:9002/HelloHeader",
+        String request =
+                "<soapenv:Envelope xmlns:soapenv=\"http://schemas.xmlsoap.org/soap/envelope/\" xmlns:spr=\"http://hello.cxf.ws.connectors.bonitasoft.org/\">"+
+                " <soapenv:Header/>"+
+                " <soapenv:Body>"+
+                "    <spr:sayHi>"+
+                "       <arg0>Rodrigue test</arg0>"+
+                "    </spr:sayHi>"+
+                " </soapenv:Body>"+
+                "</soapenv:Envelope>";
+        final String result = execute(request, SOAPBinding.SOAP11HTTP_BINDING, "http://localhost:9002/HelloHeader",
                 "HelloHeaderImplService", "HelloWorldImplPort",
                 "http://hello.cxf.ws.connectors.bonitasoft.org/", null, "guest", "guest", requestHeaders);
+        assertTrue(result.contains("Hello Rodrigue test"));
```

Comment:
❓ can we introduce assertj because for now, if the assertion fail, we have no feedback about what we have in the result
This can apply to all assertions done in this Test class

Reply:
I'll do that.

Reply:
done